### PR TITLE
Feature/template images

### PR DESCRIPTION
**Pull request by ValeriiMalkov**
Add implementation template images
on the HMI for next RPC:

    SetGlobalProperties
    AddCommand
    Show
    ShowConstantTBT
    SendLocation
    UpdateTurnList
    Alert
    ScrollableMessage
    AlertManeuver
    CreateInteractionChoiceSet

### CODE REVIEW

File: app/controller/sdl/Abstract/Controller.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ SDL.SDLController = Em.Object.extend(
      * Active application model binding type {SDLAppModel}
      */
     model: null,
+    isWarning:false,
```
Comment (yang1070) [inline app/controller/sdl/Abstract/Controller.js]: if the warning stuff related to template image?

File: app/model/sdl/MediaModel.js [hunk 2]
```diff
@@ -341,14 +354,21 @@ SDL.SDLMediaModel = SDL.ABSAppModel.extend({
...
       this.appInfo.set('alignment', 'text-align:center');
     }
-
+    
     if (params.graphic != null) {
+      var image = params.graphic.value;
+          var length=image.length;
+          str='.png';
+          var isPng=image.includes(str,length-5);
+          if(isPng){
```
Comment (yang1070) [inline app/model/sdl/MediaModel.js]: code style - indent for the next few lines

**Comment by yang1070** [comment]
@ValeriiMalkov  AddSubMenu can also has an image, that need updates too

**Comment by yang1070** [comment]
ResetGlobalProperties  has an image parameter too

**Comment by AKalinich-Luxoft** [comment]
@yang1070  `AddSubMenu` is a separate proposal, which is not merged to develop. See #88 
So most likely we can't add support for this functionality for now.

**Comment by AKalinich-Luxoft** [comment]
@yang1070 `SetGlobalProperties` / `ResetGlobalProperties` has no visual effect on HMI so there is no need to change anything fot them

File: app/controller/sdl/Abstract/Controller.js (hunk 1 above)
Comment (AKalinich-Luxoft) [reply app/controller/sdl/Abstract/Controller.js]: @yang1070 HMI should check incoming images extensions and respond with warnings if it is not PNG. However I could see some unused variables like this one.
I removed unused variables in https://github.com/smartdevicelink/sdl_hmi/pull/69/commits/e50714bdfe67ad666acc42e28f3ee037e8e43b21

File: app/model/sdl/MediaModel.js (hunk 2 above)
Comment (AKalinich-Luxoft) [reply app/model/sdl/MediaModel.js]: @yang1070 fixed in https://github.com/smartdevicelink/sdl_hmi/pull/69/commits/e50714bdfe67ad666acc42e28f3ee037e8e43b21

**Pull request merged**